internal/handlers/telegram/hander_test.go

```diff
@@ -0,0 +1,57 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+	"github.com/kyokomi/emoji"
+	"github.com/ritlug/teleirc/internal"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestStickerSmile(t *testing.T) {
+	correct := "<test>😄"
+	updateObj := tgbotapi.Update{
+		Message: &tgbotapi.Message{
+			From: &tgbotapi.User{
+				UserName: "test",
+			},
+			Sticker: &tgbotapi.Sticker{
+				Emoji: emoji.Sprint(":smile:"),
+			},
+		},
+	}
+	clientObj := &Client{
+		Settings: internal.TelegramSettings{
+			Prefix: "<",
+			Suffix: ">",
+		},
+		sendToIrc: func(s string) {
+			assert.Equal(t, correct, s)
+		},
+	}
+
+	stickerHandler(clientObj, updateObj)
+}
+
+func TestMessageRandom(t *testing.T) {
```

Comment:
Same thing with sticker message. This is only testing our positive test cases. Same thing as above, what if `UserName` field is missing, etc.

Reply:
fixed